lib/msf/core/handler/reverse_http.rb

```diff
@@ -158,25 +158,14 @@ def setup_handler
   end
 
   #
-  # Simply calls stop handler to ensure that things are cool.
```

Comment:
This is from b27c9b9efc505888e0a4190ee7d6cdf759a0ad58 when I tried to git merge upstream-master to update the branch.